mne/io/brainvision/brainvision.py

```diff
@@ -383,56 +397,92 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
     # set to zero.
     settings = settings.splitlines()
     idx = None
+
     if 'Channels' in settings:
         idx = settings.index('Channels')
         settings = settings[idx + 1:]
+        hp_col, lp_col = 4, 5
         for idx, setting in enumerate(settings):
             if re.match('#\s+Name', setting):
                 break
             else:
                 idx = None
 
+    # If software filters are active, then they override the hardware setup
+    # But we still want to be able to double check the channel names
+    # for alignment purposes, we keep track of the hardware setting idx
+    idx_amp = idx
+
+    if 'S o f t w a r e  F i l t e r s' in settings:
+        idx = settings.index('S o f t w a r e  F i l t e r s')
+        for idx, setting in enumerate(settings[idx + 1:], idx + 1):
+            if re.match('#\s+Low Cutoff', setting):
+                hp_col, lp_col = 1, 2
+                warn('Online software filter detected. Using software '
+                     'filter settings and ignoring hardware values')
```

Comment:
![psd](https://cloud.githubusercontent.com/assets/1677783/18008719/862ec02e-6ba9-11e6-9712-a4e4dd7efaca.png)

That's from a file with 0.1 Hz to 1000 Hz hardware, sampled at 250 Hz (so effective 150 Hz high cutoff) 
```
Channels
--------
#     Name      Phys. Chn.    Resolution [µV]  Low Cutoff [s]   High Cutoff [Hz]   Notch [Hz]
1     F7          1                0.1               10             1000              Off
```

but a 50 Hz high cutoff and notch in the software filters,

```
S o f t w a r e  F i l t e r s
==============================
#     Low Cutoff [s]   High Cutoff [Hz]   Notch [Hz]
1      0.9              50                 50
````

So it looks the software filters are in use.